api/v1/clusterextensionrevision_types.go

```diff
@@ -43,23 +47,54 @@ const (
 )
 
 // ClusterExtensionRevisionSpec defines the desired state of ClusterExtensionRevision.
+//
+// A ClusterExtensionRevision represents a specific immutable snapshot of the objects
+// to be installed for a ClusterExtension. Each revision is rolled out in phases,
+// with objects organized by their Group-Kind into well-known phases such as namespaces,
+// rbac, crds, and deploy.
 type ClusterExtensionRevisionSpec struct {
-	// Specifies the lifecycle state of the ClusterExtensionRevision.
+	// The lifecycleState field specifies the lifecycle state of the ClusterExtensionRevision.
+	//
+	// When set to "Active" (the default), the revision is actively managed and reconciled.
+	// When set to "Paused", reconciliation is disabled but status updates continue.
```

Comment:
Do you want to drop it, given that it is not implemented and reintroduce it when needed?